controller/volume_attachment_controller.go

```diff
@@ -618,17 +610,7 @@ func (vac *VolumeAttachmentController) shouldDoDetach(va *longhorn.VolumeAttachm
 func hasUninterruptibleTicket(attachmentTickets map[string]*longhorn.AttachmentTicket) bool {
 	for _, ticket := range attachmentTickets {
 		if ticket.Type != longhorn.AttacherTypeSnapshotController &&
-			ticket.Type != longhorn.AttacherTypeBackupController &&
-			ticket.Type != longhorn.AttacherTypeVolumeRebuildingController {
-			return true
-		}
-	}
-	return false
-}
-
-func hasInterruptibleAndFrontendDisabledTicket(attachmentTickets map[string]*longhorn.AttachmentTicket) bool {
-	for _, ticket := range attachmentTickets {
-		if ticket.Type == longhorn.AttacherTypeVolumeRebuildingController {
```

Comment:
Is it the cleanup from the previous `rebuilding-controller` feature?

Reply:
>Is it the cleanup from the previous rebuilding-controller feature?

Did you mean the volume rebuild VA ticket will be deleted by `volume-rebuild-controler`?

Reply:
The volume rebuild VA ticket should be able to be preempted by the higher-priority VA ticket.

Reply:
Sorry for the confusion. I meant we removed this logic because it is no longer needed and it is the leftover from the original v2 rebuilding feature?

Reply:
>because it is no longer needed and it is the leftover from the original v2 rebuilding feature?

I think Yes.